composer.json

```diff
@@ -36,12 +36,12 @@
         "twig/extensions": "^1.4",
         "beberlei/DoctrineExtensions": "^1.0",
         "knplabs/knp-menu-bundle": "^2.0",
-        "tetranz/select2entity-bundle": "^2.9"
+        "tetranz/select2entity-bundle": "^2.9",
+        "symfony/web-server-bundle": "^3.3"
     },
     "require-dev": {
         "sensio/generator-bundle": "^3.0",
         "symfony/phpunit-bridge": "^3.0",
-        "symfony/web-server-bundle": "^3.3",
```

Comment:
We really do not want the web-server-bundle in production. It should only be in require-dev, because we use a completely different web server for production websites.

Reply:
oh sorry sorry, you told me this before. I had to put it there to run the app for some reason, then forgot to change back.. Solved!

Reply:
The web-server-bundle is still in the wrong place. Maybe you need to push a commit or something.

Reply:
should be ok now (hopefully..)